### PR TITLE
Luxury Medipens now have lavaland extract in them

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Luxury medipens cost double the price of the emergency one, and all it really does is punish you for taking damage rather than healing it, hopefully the 2 units of lavaland extract will help it heal rather than kill the user.

## Why It's Good For The Game

We're a beginner friendly server punishing beginners for using things that are supposed to help them?

## Changelog
:cl:
add: Lavaland extract
balance: Luxury medipens now have 2 units of lavaland extract in them
/:cl:

### CODE REVIEW

**Comment by GuillaumePrata** [comment]
The last thing Shaft Miners need is a buff to help em kill Megafauna.

**Comment by GuillaumePrata** [comment]
Here are the stats of the chemicals on the Luxury Medipen and how much they heal:
Salbutamol = Procs 100 times, each proc heals 3 Oxy damage for 300 in total.
Penthrite = Procs 50 times, each procs heals 2 Brute/Burn/Tox and 6 Oxy damage for a total of 100 B/B/T and 300 Oxy. Also increases your max HP by 60% basically
Oxadralone = Procs 50 times, each procs heals 4 burn if burn damage is above 25 or 0.5 if it's not for a total of 200 or 25
Salicid Acid = Procs 50 times, each procs heals 4 brute if brute damage is above 25 or 0.5 if it's not for a total of 200 or 25
Omnizine = Procs 100 times, each procs heals 0.5 of B/B/T/O for a total of 50 of each
Leporazine = Procs 25 times, each proc stabilize the patient temperature
	Total healed
	Brute = 350 or 175
	Burn  = 350 or 175
	Tox   = 150
	Oxy   = 650

The normal survival heals a total of 80 burn and 120 brute and doesn't increase your max HP.
This lavaland extract change will add a total of 25 brute and burn healing to the luxury medipen.
Comparing both of those is kinda silly, the luxury one is already more than double the healing of the basic one and well worth it's price before any change is made to it.

**Comment by JohnFulpWillard** [comment]
> 
> 
> Here are the stats of the chemicals on the Luxury Medipen and how much they heal:
> Salbutamol = Procs 100 times, each proc heals 3 Oxy damage for 300 in total.
> Penthrite = Procs 50 times, each procs heals 2 Brute/Burn/Tox and 6 Oxy damage for a total of 100 B/B/T and 300 Oxy. Also increases your max HP by 60% basically
> Oxadralone = Procs 50 times, each procs heals 4 burn if burn damage is above 25 or 0.5 if it's not for a total of 200 or 25
> Salicid Acid = Procs 50 times, each procs heals 4 brute if brute damage is above 25 or 0.5 if it's not for a total of 200 or 25
> Omnizine = Procs 100 times, each procs heals 0.5 of B/B/T/O for a total of 50 of each
> Leporazine = Procs 25 times, each proc stabilize the patient temperature
> Total healed
> Brute = 350 or 175
> Burn = 350 or 175
> Tox = 150
> Oxy = 650
> 
> The normal survival heals a total of 80 burn and 120 brute and doesn't increase your max HP.
> This lavaland extract change will add a total of 25 brute and burn healing to the luxury medipen.
> Comparing both of those is kinda silly, the luxury one is already more than double the healing of the basic one and well worth it's price before any change is made to it.

Even if it heals a lot, that healing is extremely slow, which is the opposite of what a medipen should be. You need quick bursts of healing to fight megafauna, not overtime. When you're fighting megafauna, the moment you hit -60% health, you're gone for good, unlike the pre-nerf regular medipen which would've quickly healed you out of it so you wouldn't GET to -60%.

**Comment by ErwinSnowden** [comment]
git gud

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -241,7 +241,7 @@
 	inhand_icon_state = "atropen"
 	volume = 60
 	amount_per_transfer_from_this = 60
```
Comment (ATH1909) [inline code/modules/reagents/reagent_containers/hypospray.dm:243]: you forgot to increase these values to accommodate for the new chem

**Comment by ATH1909** [comment]
> > Here are the stats of the chemicals on the Luxury Medipen and how much they heal:
> > Salbutamol = Procs 100 times, each proc heals 3 Oxy damage for 300 in total.
> > Penthrite = Procs 50 times, each procs heals 2 Brute/Burn/Tox and 6 Oxy damage for a total of 100 B/B/T and 300 Oxy. Also increases your max HP by 60% basically
> > Oxadralone = Procs 50 times, each procs heals 4 burn if burn damage is above 25 or 0.5 if it's not for a total of 200 or 25
> > Salicid Acid = Procs 50 times, each procs heals 4 brute if brute damage is above 25 or 0.5 if it's not for a total of 200 or 25
> > Omnizine = Procs 100 times, each procs heals 0.5 of B/B/T/O for a total of 50 of each
> > Leporazine = Procs 25 times, each proc stabilize the patient temperature
> > Total healed
> > Brute = 350 or 175
> > Burn = 350 or 175
> > Tox = 150
> > Oxy = 650
> > The normal survival heals a total of 80 burn and 120 brute and doesn't increase your max HP.
> > This lavaland extract change will add a total of 25 brute and burn healing to the luxury medipen.
> > Comparing both of those is kinda silly, the luxury one is already more than double the healing of the basic one and well worth it's price before any change is made to it.
> 
> Even if it heals a lot, that healing is extremely slow, which is the opposite of what a medipen should be. You need quick bursts of healing to fight megafauna, not overtime. When you're fighting megafauna, the moment you hit -60% health, you're gone for good, unlike the pre-nerf regular medipen which would've quickly healed you out of it so you wouldn't GET to -60%.

I'm not a miner, but don't megafauna continue to kick you while you're down/critted anyway?

**Comment by E231kei500bandai** [comment]
> Even if it heals a lot, that healing is extremely slow, which is the opposite of what a medipen should be. You need quick bursts of healing to fight megafauna, not overtime. When you're fighting megafauna, the moment you hit -60% health, you're gone for good, unlike the pre-nerf regular medipen which would've quickly healed you out of it so you wouldn't GET to -60%.

it has the fastest healing chems in the game what you talking about it heals 4.5 brute and burn per tick theres nothing faster than that

**Pull request closed without merging**